pkg/test/common.go

```diff
@@ -0,0 +1,20 @@
+package test
```

Comment:
could we figure out a better name for this file? I'd like to avoid `pkg/test/common.go` since it can contain anything and can grow to be a _god file_. How about `api_types.go`?

Reply:
I also don't like the `common.go` name, but couldn't figure out the more appropriate name earlier. Changed to `api_types.go` as suggested though

Reply:
addressedaddressed